modules/auxiliary/scanner/oracle/tnspoison_checker.rb

```diff
@@ -42,8 +42,19 @@ def run_host(ip)
       send_packet = tns_packet("(CONNECT_DATA=(COMMAND=service_register_NSGR))")
       sock.put(send_packet)
       packet = sock.read(100)
+      hex_packet = Rex::Text.to_hex(packet, prefix = ':')
```

Comment:
Implemented IF-ELSE block to handle "nil" value of "packet" variable. It should fix the stack trace bug. Latest update to code fixes this issue.